janitor.Rproj

```diff
@@ -0,0 +1,17 @@
+Version: 1.0
```

Comment:
Oh, really? You should definitely start using projects for all of your work. If you are using RStudio, it should be a standard part of your workflow. Basic intro here https://support.rstudio.com/hc/en-us/articles/200526207-Using-Projects

Could you accept https://github.com/sfirke/janitor/pull/79? This will make development easier for me.